eng/scripts/InstallationCheck.ps1

```diff
@@ -0,0 +1,32 @@
+param (
+    [Parameter()]
+    [string] $ArtifactsDirectory,
+
+    [Parameter()]
+    [string] $Artifact,
+
+    [Parameter()]
+    [string] $PipelineWorkspace
+)
+
+Write-Host "dotnet new console"
+dotnet new console
+$localFeed = "$PipelineWorkspace/$ArtifactsDirectory-signed/$Artifact"
+Write-Host "dotnet nuget add source $localFeed"
+dotnet nuget add source $localFeed
+
+$version = (Get-ChildItem "$localFeed/*.nupkg" -Exclude "*.symbols.nupkg" -Name).replace(".nupkg","").replace("$Artifact.","")
+Write-Host "dotnet add package $Artifact --version $version --no-restore"
+dotnet add package $Artifact --version $version --no-restore
+if ($LASTEXITCODE) {
+    exit $LASTEXITCODE
+}
+
+Write-Host "dotnet nuget locals all --clear"
+dotnet nuget locals all --clear
```

Comment:
If you use `--packages` to isolate the packages you shouldn't need to call this clear.

Reply:
Don't we still need to clear the cache? (especially when doing the retries here https://github.com/Azure/azure-sdk-for-net/issues/23739

Reply:
When you use an independent packages folder it should only uses packages from that path. For retries you can just delete that directory.

Reply:
btw ill just push those changes along with PR for this https://github.com/Azure/azure-sdk-for-net/issues/23739